### ISSUE TITLE
Remove Alpha tag for Useful Links component

### ISSUE THREAD

**Issue by j-pendlebury** [BBC-archive/psammead#2254]
**Is your feature request related to a problem? Please describe.**
In https://github.com/bbc/psammead/pull/2236, the useful links component is being built as an alpha component. 

This component should be amended and tested according to the a11y requirements.

**Describe the solution you'd like**
Remove alpha tag after successful a11y swarm on the component.

**Testing notes**
A11y testing required.

- [ ] This feature is expected to need manual testing.

**Additional context**
WSFRONTPGS-226

**Comment by DenisHdz** [comment]
Blocked on https://github.com/bbc/psammead/issues/2275

**Comment by OlgaLyubin** [comment]
Unblocked on https://github.com/bbc/psammead/issues/2275
